### DISCUSSION TITLE
Opening two urls on a new window using different tabs

### DISCUSSION THREAD

**Discussion by pinarruiz** [qutebrowser/qutebrowser#7363] [Ideas] [answered]
When using a command like: `qutebrowser --target window qutebrowser.org debian.org` the two urls open on different windows, i dont know if there is a way of changing this behabiour, but since i havent found one i will suggest using a new target like window-tabs, so `qutebrowser --target window-tabs qutebrowser.org debian.org` will create a new window with this two urls opened as tabs.

What do you think?

**Comment by emanuele6** [comment, marked as answer]
You can get that behaviour now with: `qutebrowser --target window qutebrowser.org ':open -t debian.org'` (not 100% sure if that is intended to be honest), but yeah `window-tabs` may be nice. :)

**Reply by pinarruiz** [reply]
Yeah, i tried that, but it seemed a little bit of a workaround instead of an actual solution, for the time being i will use that, but i will open a FR to add the new target.
